roles/ddns_updater/defaults/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# enable the role
+ddns_updater_enabled: false
+
+# Data directory for config file
+ddns_updater_data_directory: "{{ docker_home }}/ddns_updater"
+
+# enable in order to make web ui reachable from external
+ddns_updater_available_externally: "false"
+
+# Port for the web ui
+ddns_updater_port: 8202
+ddns_updater_hostname: "ddns"
+
+# User to run the container with
+ddns_updater_user_id: "0"
+
+# Token to be used to send messages via gotify
+# ddns_updater_gotify_token: "ABCDEFGHIJKLMOP"
+
+# url of gotify server
+gotify_url: "gotify.{{ ansible_nas_domain }}"
```

Comment:
in order to avoid having to define `gotify_url` for every application that wants to send to gotify we probably have to define this variable in all.yml as well